mobile/src/common/rounded-avatar.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import { View, Image } from 'react-native';
+
+export interface Props {
```

Comment:
rename Props please to something like RoundedAvatarProps. Otherwise it'll be harder to find this type or export it (what if different file exports this Props and something called Props from another file?)